src/semeio/forward_models/design_kw/design_kw.py

```diff
@@ -178,3 +184,41 @@ def rm_genkw_prefix(paramsdict, ignoreprefixes="LOG10_"):
         for keyvalue in keyvalues
         if keys.count(keyvalue[0]) == 1
     }
+
+
+def validate_configuration(
+    template_file_name: str, parameters_file_name: str = "parameters.txt"
+) -> None:
+    try:
+        if not os.path.exists(template_file_name):
+            raise ValueError(
+                f"Argument template_file: '{template_file_name}' is not an existing file"
+            )
+
+        if not os.path.exists(parameters_file_name):
+            raise ValueError(
+                f"Could not find required parameters file from DESIGN2PARAMS: '{parameters_file_name}'"
+            )
+        with open(parameters_file_name, encoding="utf-8") as parameters_file:
+            parameters = parameters_file.readlines()
+
+        key_vals = extract_key_value(parameters)
```

Comment:
extract_key_value currently raises `ValueError` from shlex with the message "No closing quotation" see https://github.com/equinor/semeio/issues/641 . `extract_key_value` needs to be fixed so that it doesn't raise in order for this design to be used. I would suggest both fixing the bug with `extract_key_value` so that it shows a reasonable error message rather than throwing from shlex, and not relying on catching an exception in order to indicate warnings.